using_images/s2i_images/java.adoc

```diff
@@ -13,49 +13,50 @@ toc::[]
 [[s2i-images-java-overview]]
 == Overview
 
-{product-title} provides an
-xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[S2I
-builder image] for building Java applications.  This builder image takes your
-application source or binary artifacts, builds the source using Maven (if source
-was provided), and assembles the artifacts with any required dependencies to
-create a new, ready-to-run image containing your Java application. This
-resulting image can be run on {product-title} or run directly with Docker.
-
-The builder image is intended for use with
+{product-title} provides
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[S2I builder images] for building Java applications. These builder images take your application source or binary artifacts, build the source using Maven (if the source was provided), and assemble the artifacts with any required dependencies to create a new, ready-to-run image containing your Java application. This resulting image can be run on {product-title} or run directly with Docker.
+
+The builder images are intended for use with
 link:https://maven.apache.org[Maven]-based Java standalone projects that are run
 via main class.
 
 
 [[s2i-images-java-versions]]
 == Versions
 
-The current version of the Java S2I builder image supports OpenJDK 1.8, Jolokia
-1.3.5, and Maven 3.3.9-2.8.
+The current version of the Java S2I builder images support OpenJDK 1.8 and 11,
+Jolokia 1.6.2, and Maven 3.6.
 
 
 [[s2i-images-java-images]]
 == Images
 
-The RHEL 7 image is available through the Red Hat Registry:
+The RHEL 7 and RHEL 8 images are available through the Red Hat Registry:
+
+*RHEL 7 based images*
 
 ----
 $ docker pull registry.redhat.io/redhat-openjdk-18/openjdk18-openshift
+$ docker pull registry.redhat.io/openjdk/openjdk-11-rhel7
+----
+
+*RHEL 8 based images*
+
+----
+$ docker pull registry.redhat.io/ubi8/openjdk-8
+$ docker pull registry.redhat.io/ubi8/openjdk-11
 ----
 
 ifdef::openshift-online[]
 You can use the image through the `redhat-openjdk18-openshift` image stream.
```

Comment:
@jmtd Should this line be updated to also include the other images? If so, what are the names of the other image streams?

Reply:
From what I can see they're all accounted for now: 2 RHEL8-based, and 2 RHEL7-based.

Reply:
I was referring to this line: "You can use the image through the `redhat-openjdk18-openshift` image stream." Should I update this line to include the other images? If so, what stream would be used for the other images?

Reply:
I think yes, we should add other ImageStream names. I'm just now trying to verify what they are for OpenShift Online.

Reply:
The following ImageStreams are valid in OpenShift Online:

 * redhat-openjdk18-openshift (RHEL7, JDK8)
 * ubi8-openjdk-11 (RHEL UBI8, JDK11)
 * ubi8-openjdk-8 (RHEL UBI8, JDK8)
 * openjdk-11-rhel7 (RHEL7, JDK11)
 * java
 
 "java" is a catch-all ImageStream which incorporates all of the other ones, and unless otherwise specified (via a tag), defaults to RHEL8/JDK11.